.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,99 @@
+
+name: "Continuous Integration"
```

```suggestion
name: "CI"
```

The matrix names will be long enough already. Rename the file accordingly as well.